bookshelf-standard/2-structured-data/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2016 Google Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project>                               <!-- REQUIRED -->
+
+  <modelVersion>4.0.0</modelVersion>    <!-- REQUIRED -->
+  <packaging>war</packaging>            <!-- REQUIRED -->
+
+  <groupId>com.example.standard.gettingstarted</groupId>
+  <artifactId>bookshelf-standard-2</artifactId>    <!-- Name of your project -->
+  <version>1.0-SNAPSHOT</version>       <!-- xx.xx.xx -SNAPSHOT means development -->
+
+  <parent> <!-- Only used for testing - NOT REQUIRED -->
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.1</version>
+  </parent>
+
+  <properties>
+    <!-- [START config] -->
+    <bookshelf.storageType>datastore</bookshelf.storageType>   <!-- datastore or cloudsql -->
+
+    <sql.dbName>bookshelf</sql.dbName>                        <!-- A reasonable default -->
+<!-- Instance Connection Name - project:region:dbName -->
+<!-- -Dsql.instanceName=localhost to use a local MySQL server -->
+    <sql.instanceName>Your database connectionName here. See `gcloud sql instances describe [instanceName]`</sql.instanceName>
```

Comment:
I'm not sure what you mean? Where should I add hyphens, and what does it make easier to click?